plugin/api.go

```diff
@@ -152,6 +152,9 @@ type API interface {
 	// GetChannelMembers gets a channel membership for all users.
 	GetChannelMembers(channelId string, page, perPage int) (*model.ChannelMembers, *model.AppError)
 
+	// GetChannelMembersByIds gets a channel membership for a particular User
```

Comment:
Can you document the minimum version required here?

```suggestion
	// GetChannelMembersByIds gets a channel membership for a particular User
	//
	// Minimum server version: 5.6
```